### DISCUSSION TITLE
el-table 嵌套dialog 显示bug

### DISCUSSION THREAD

**Discussion by chenzhongqing666** [element-plus/element-plus#7691] [General]
在el-table行内嵌套el-dialog时会出现层级问题
![image](https://user-images.githubusercontent.com/66964903/168407960-5ac13c65-447a-4c1e-8df8-d9d037bdc400.png)

**Comment by msidolphin** [comment]
挂载到表格外

**Reply by chenzhongqing666** [reply]
挂载到表格外确实能够解决问题，但是这个bug确实是存在的
